README.md

```diff
@@ -1 +1,58 @@
 # book-store-react
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is a website that allows users to add/remove books from a list. This project is build using JavaScript objects and arrays.
+
+
+## Built With
+
+- HTML
+- CSS 
+- Javascript
+- React
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Clone repository in your local machine 
+- cd Portfolio
+- open `index.html` in your browser.
+- You can use live server.
+
+## Live Demo
+
+[Books Live](https://basirking.github.io/Awesome-Book/)
+
+## Author
+
+👤 **Basir Mohammadi**
+
+- GitHub: [@basirking](https://github.com/Basir-Mohammadi)
+- LinkedIn: [Basir Mohammadi](https://www.linkedin.com/in/basirahmad1312/)
+
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/Basir-Mohammadi/Awesome_Books/issues](https://github.com/Basir-Mohammadi/book-store-react/issues)
+
+## Show your support
+
+Give a ⭐️ to if you like this project!
+
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- Please, kindly ensure the MIT.md licensed content are available and not a broken link to make your readme more professional.